### PR TITLE
fix: Welcoming grpc to the 21st century

### PR DESCRIPTION
**Pull request by evenius**
This should solve a couple of issues I've had with grpc web and vite specifically, now we generate 100% typescript, and most of the callback chains are replaces with UnaryCalls/Promises :)

### CODE REVIEW

File: src/controllers/components/emulator/net/JsepProtocol.ts [hunk 1]
```diff
@@ -76,7 +77,7 @@ export default class JsepProtocol {
     this.eventForwarders = {};
 
     if (this.stream) {
-      this.stream.cancel();
+      // this.stream.cancel();
```
Comment (AntonNeld) [inline src/controllers/components/emulator/net/JsepProtocol.ts:80]: Is this intentional or a debug leftover? If intentional, maybe better to remove it or add a more elaborate comment.

File: src/controllers/proto/emulator_web_client.ts [hunk 2]
```diff
@@ -13,8 +13,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { EmulatorControllerClient } from '../proto/Emulator_controllerServiceClientPb';
-import { RtcClient } from '../proto/Rtc_serviceServiceClientPb';
+// import type { RpcTransport } from '@protobuf-ts/runtime-rpc';
```
Comment (vcs222) [inline src/controllers/proto/emulator_web_client.ts:16]: remove old imports

**Review by vcs222 — APPROVED** [review]

**Review by kouvaliasnick — APPROVED** [review]

**Pull request merged**